drivers/python/rethinkdb/asyncio_net/net_asyncio.py

```diff
@@ -71,6 +71,26 @@ def __init__(self, *args, **kwargs):
         Cursor.__init__(self, *args, **kwargs)
         self.new_response = asyncio.Future()
 
+    @asyncio.coroutine
+    def __aiter__(self):
+        return self
+
+    @asyncio.coroutine
+    def __anext__(self):
+        data = self._get_next(None)
```

Comment:
I think this should be `data = yield from self._get_next(None)` since self._get_next() returns a coroutine and the `if data` below would always evaluate as True.